tools/runner

```diff
@@ -54,6 +54,9 @@ new_path = [os.path.abspath(dirs['out'] + "/runners/bin/"), os.environ['PATH']]
 
 os.environ['PATH'] = ":".join(new_path)
 
+if not runner_obj.can_run():
+    sys.exit(0)
```

Comment:
How does this resolve #83? 

To actually solve #83 I think that we should exclude the runner from the report completely, not "fake-pass" all the tests.

Additionally I don't think it should be quiet. We should at least show a warning about skipping a runner.

Thirdly I think there still should be a way to force-require all runners (could be a Makefile flag). We don't want to exclude anything in CI by accident.

Reply:
By doing a "fake-pass" at this stage no log file is generated and thus the column for a particular tool won't be included in the report.

Detecting available tools could be moved to main Makefile and we could handle the warnings and hard requirement for all runners there

Reply:
That makes sense. But still - at least a part of it should be moved to makefile (to be able to error on this in CI).

I am also wondering if we could somehow not invoke the runners that fail on can run (am I right to think that this will be executed for each test anyway? We should be able to check it just once).